### PR TITLE
packet support in eden commands

### PR DESCRIPTION
**Pull request by giggsoff**
support for eden packet commands to run/delete servers on Equinix platform.
also this PR adds `ipxe-override` flag to override lines in ipxe config.

### CODE REVIEW

**Review by rvs — COMMENTED** [review]
Hey @giggsoff -- sorry to come to this late, but it *really* needs to be documented. Can you please submit a separate PR with docs update?

File: cmd/edenSetup.go [hunk 1]
```diff
@@ -49,6 +49,8 @@ var (
 
 	softserial    string
 	zedcontrolURL string
+
+	ipxeOverride string
```
Comment (rvs) [inline cmd/edenSetup.go:53]: I am not sure I'm convinced this is the best way to handle this. Why not just separate ipxe config files to begin with @giggsoff ?

Comment (giggsoff) [reply cmd/edenSetup.go:53]: Agree, not the best, tried maps and slices from cobra implementation, but the first one do not support spaces in lines, the second one do not support comma (and I need exactly comma to use zfs).
Can you please elaborate, what you mean by separation of ipxe config? Separation of generation and uploading into eserver?

Comment (rvs) [reply cmd/edenSetup.go:53]: I mean -- why bother with replacing parts of the config and not just have self-container iPXE configs instead @giggsoff ?

Comment (giggsoff) [reply cmd/edenSetup.go:53]: I use something like this (sure, it is not perfect, but works) now on Jenkins (I have a plan to present some files for Jenkins in the repo as samples):
```
./eden --config eden-packet setup --netboot=true --ipxe-override "set installer_args||set installer_args eve_persist_disk=sda,"
```
It helps me to run EVE with zfs

**Pull request merged**